lib/subscriptions.js

```diff
@@ -1,71 +1,48 @@
-var subscriptions = {}
+
+var uuid = require('uuid')
 
 function Subscription(options) {
-  this.id = Math.floor(Math.random() * 1000000000).toString()
+  this.id = options.id || uuid.v4()
   this.notify_url = options.notify_url
   this.pos = options.pos || '0'
   this.filter = options.filter || {}
-  this.setTTL(options.ttl || 60 * 60 * 24)
+  this.ttl = options.ttl || 60 * 60 * 24
```

Comment:
WHAT?! I can't hear you over this awesome [track](https://www.youtube.com/watch?v=dQw4w9WgXcQ)